src/Microsoft.CodeAnalysis.NetAnalyzers/Microsoft.CodeAnalysis.NetAnalyzers.md

```diff
@@ -0,0 +1,207 @@
+
+Sr. No. | Rule ID | Title | Category | Enabled | CodeFix | Description |
```

Comment:
❔ Was this supposed to be copied from somewhere?

Reply:
This gets auto-generated for all the analyzer NuGet projects in the solutions.